ext/node/polyfills/process.ts

```diff
@@ -426,7 +427,17 @@ class Process extends EventEmitter {
   _exiting = _exiting;
 
   /** https://nodejs.org/api/process.html#processexitcode_1 */
-  exitCode: undefined | number = undefined;
+  get exitCode() {
+    return globalProcessExitCode;
+  }
+
+  set exitCode(code: number | undefined) {
+    globalProcessExitCode = code;
+    code = (typeof code === "string" ? parseInt(code) : code) ?? 0;
+    if (typeof code === "number" && !isNaN(code)) {
+      ops.op_set_exit_code(code);
+    }
```

Comment:
```suggestion
    code = parseInt(code) || 0;
    if (!isNaN(code)) {
      ops.op_set_exit_code(code);
    }
```

Handles floating point numbers too